test/integration/client-side-encryption/client_side_encryption.spec.test.ts

```diff
@@ -80,5 +81,8 @@ describe('Client Side Encryption (Legacy)', function () {
 });
 
 describe('Client Side Encryption (Unified)', function () {
+  if (process.version.includes('18')) {
+    setDefaultResultOrder('ipv4first');
+  }
```

Comment:
Should this be in a before and then reset inside an after?

Reply:
sure, done